ql/src/java/org/apache/hadoop/hive/ql/plan/Statistics.java

```diff
@@ -220,14 +220,14 @@ public Statistics clone() {
   }
 
   public void addBasicStats(Statistics stats) {
-    dataSize += stats.dataSize;
+    dataSize = StatsUtils.safeAdd(dataSize,stats.dataSize);
```

Comment:
NIT, I think we miss a space between the arguments.

Reply:
Thank you for your feedback, and I apologize for my oversight. I've been busy with work these past few days and forgot to check GitHub. I will make the changes right after I finish work.